newIDE/app/src/EventsFunctionsList/index.js

```diff
@@ -37,7 +37,17 @@ export type EventsFunctionCreationParameters = {|
 |};
 
 const getEventsFunctionName = (eventsFunction: gdEventsFunction) =>
-  eventsFunction.getName() + (eventsFunction.isPrivate() ? ' (private)' : '');
+  eventsFunction.isPrivate() ? (
+    <React.Fragment>
+      <VisibilityOffIcon
+        fontSize="small"
+        style={{ marginRight: 5, verticalAlign: 'bottom' }}
+      />
+      {eventsFunction.getName()}
+    </React.Fragment>
+  ) : (
+    <>{eventsFunction.getName()}</>
+  );
```

Comment:
No need for a fragment here? A text can always be used as a react child directly.
```suggestion
  ) : eventsFunction.getName();
```

Reply:
This will resolve the title issue when the function is public.